### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by Escanord**
Complete week1 work for SPS with the main objective of sketching frontend for portfolio website. I add the following main sections to the webpage to better illustrate myself:
- Education
- Technical Skills
- Experiences
- Projects
- Awards

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,12 +8,126 @@
...
+              <li><b>GPA:</b> 4.0/4.0 - Dean’s High Honor List. </li>
+              <li><b>Undergrad Courses:</b> Data Structures & Algorithms, Software Engineering, Web Development, OS.</li>
+              <li><b>Graduate Courses:</b> Database Systems, Theoretical Statistics.</li>
+          </ul>
+      </div>
+
+      <h2>Technical Skills</h2>
+      <div>
+          <ul>
+              <li> <b>Languages:</b> Python, Java, JavaScript, C/C++, SQL, Cypher, XQuery, SPARQL</li>
```
Comment (sarahrittgers26) [inline portfolio/src/main/webapp/index.html:37]: Nit: for consistency with the above, can we remove the extra space before ‘<b>’? (Or add the space above)

**Review by sarahrittgers26 — APPROVED** [review]
Nice job!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (Escanord) [reply portfolio/src/main/webapp/index.html:37]: I'm not sure if I fully understand what you mean or which line you mention. Can you elaborate a little more?

Comment (sarahrittgers26) [reply portfolio/src/main/webapp/index.html:37]: Sorry, had a typo from autocorrect :)

Let’s remove the space before the ‘<b>’ on line 37 and below, for consistency

**Pull request merged**